### PR TITLE
API: Fix /api/modules

### PR DESCRIPTION
**Pull request by Pallabi-ghosh**
# Description

There was a bug in this module file and I fixed it. 

Fixes # (149)

# How to test?

I tested it in swagger and also run the database. 

# Checklist

- [ X ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/server/api/controllers/modules.controller.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const moment = require('moment-timezone');
 
 const getModules = async () => {
   try {
-    return await knex('modules').select('modules.id', 'modules.title');
+    return await knex('modules').select('modules.id');
```
Comment (tpetrina) [inline src/server/api/controllers/modules.controller.js:10]: Just id is not interesting - we need to get the module's name out as well.

File: src/server/api/controllers/modules.controller.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ const getModules = async () => {
 const getModuleById = async (id) => {
   try {
     const modules = await knex('modules')
-      .select('modules.id as id', 'title')
+      .select('modules.id as id')
```
Comment (tpetrina) [inline src/server/api/controllers/modules.controller.js:19]: Same as above - just id is not really useful.

**Review by tpetrina — CHANGES_REQUESTED** [review]

File: src/server/api/controllers/modules.controller.js (hunk 1 above)
Comment (Pallabi-ghosh) [reply src/server/api/controllers/modules.controller.js:10]: In the migration file there is no column called `title`. What should be done?

File: src/server/api/controllers/modules.controller.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ const moment = require('moment-timezone');
 
 const getModules = async () => {
   try {
-    return await knex('modules').select('modules.id', 'modules.title');
+    return await knex('modules').select('modules.id', 'modules.module_name');
```
Comment (tpetrina) [inline src/server/api/controllers/modules.controller.js:10]: Return as `name` like you did below or as `moduleName`. Between the different endpoints, you need to be consistent.

**Review by tpetrina — CHANGES_REQUESTED** [review]
Please use the same property name in all endpoints. Choose either `name` or `moduleName`. Just not all three (including `module_name`).

**Review by bytewiz — APPROVED** [review]
Seems good to me 👍

**Review by nuha90 — APPROVED** [review]

**Review by tpetrina — APPROVED** [review]

**Pull request merged**